### ISSUE TITLE
Location of twitter message (tweet) in a map

### ISSUE THREAD

**Issue by nidele** [ushahidi/platform#815]
I'm testing ushahidi.io but i note that if i set a twitter application the tweet (with gps data and hashtag) compare in draft without location. i think this function should be usefull!

**Comment by hnykda** [comment]
This could be easily plugged in by firstly implementing this #660 . (maybe there can be even localization based on IP in mail header)

**Comment by copystar** [comment]
Just wanted to note that when I try to bring in tweets into my map I get this error:  "Validation Error
values.location_default must be an array of lat and lon "

**Comment by jshorland** [comment]
Thanks for logging @copystar. Could you give us steps to reproduce the error?

**Comment by copystar** [comment]
The steps were. First, I wrote a post with the triggering tag https://twitter.com/copystar/status/720598712486084608. Then I logged into the web interface https://bikerackswindsor.ushahidi.io/posts/4/edit to make the post live. All the information came through with the exception of the location data. When I changed the option "Who can see this" I got this popup message: Validation Error values.location_default must be an array of lat and lon '
